tests/vfs_default/main.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2022 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for vfs_default
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "shell.h"
+#include "shell_commands.h"
+
+int main(void)
+{
+    /* start the shell */
```

Comment:
In light of making more features accessible to HIL tests, would it make sense to enumerate mounted file systems here, and compare that, in a test python script, to the equivalent of `grep VFS_AUTO_MOUNT ../../boards/${BOARD}/board.c | cut -d, -f3`?

(Granted, the current `TEST_BOARDS_AVAILABLE:="esp32-wroom-32 samr21-xpro"` all have no external memory, but still it'd make it easier for testers to just run `make -C tests/vfs_default BOARD=mine all flash test` and know it's good without really checking what it does).

Reply:
I added a printout of the mounts, still using `vfs_iterate_mounts`

Reply:
Now `vfs_iterate_mount_dirs()` is used.